web_src/css/repository.css

```diff
@@ -2012,31 +2012,17 @@
   margin-top: -3px;
 }
 
-.repository.settings.branches .protected-branches .selection.dropdown {
-  width: 300px;
-}
-
-.repository.settings.branches .protected-branches .item {
-  border: 1px solid var(--color-secondary);
-  padding: 10px 15px;
-}
-
-.repository.settings.branches .protected-branches .item:not(:last-child) {
-  border-bottom: 0;
-}
-
 .repository.settings.branches .branch-protection .help {
   margin-left: 26px;
   padding-top: 0;
 }
 
 .repository.settings.branches .branch-protection .fields {
-  margin-left: 20px;
   display: block;
 }
 
-.repository.settings.branches .branch-protection .whitelist {
-  margin-left: 26px;
+.repository.settings.branches .branch-protection .whitelist, #statuscheck_contexts_box {
```

Comment:
```suggestion
.repository.settings.branches .branch-protection .whitelist,
#statuscheck_contexts_box {
```

Reply:
I do not think we should add `#statuscheck_contexts_box` here.

They should share common styles.

Reply:
This is for the status list to be aligned with the checkbox above, but I also think table is not a good idea, what about reverting back to the commit before table and add some padding to this #statuscheck_contexts_box div, but I am not sure what does the “share common style” mean.

Reply:
I will try to do some changes for the table

Reply:
Done in aafbb0a7639f90a30779e0a6bfbc97108b93b3a6

More unnecessary styles are removed.

Reply:
Thanks.